src/main/java/io/pravega/perf/PerfStats.java

```diff
@@ -80,6 +80,10 @@ public PerfStats(String action, int reportingInterval, int messageSize, String c
      * Private class for start and end time.
      */
     final private class QueueProcessor implements Callable {
+        final private static int NS_PER_MICRO = 1000;
```

Comment:
We don't need many of these constants, please simplify.

Reply:
actually it reduced now  last commit "code cleanup"
ok, lets see the usages :
       final private static int NS_PER_MICRO = 1000;        /* required for NS_PER_MS */
        final private static int MICROS_PER_MS = 1000;     /* required for NS_PER_MS*/
        final private static int NS_PER_MS = NS_PER_MICRO * MICROS_PER_MS;   /* required in further code to calculate the appropriate iterations */
        final private static int PARK_NS = NS_PER_MICRO;   /* I want to keep this as it is for future fine tuning for kafka */